openvmm/hvlite_entry/src/lib.rs

```diff
@@ -802,7 +802,13 @@ fn vm_config_from_command_line(
         };
     }
 
-    if opt.vtl2 && with_hv {
+    let openhcl_vtl = if opt.vtl2 {
+        DeviceVtl::Vtl2
+    } else {
+        DeviceVtl::Vtl0
+    };
+
+    if (opt.vtl2 || opt.openhcl_devices) && with_hv {
```

Comment:
Can we combine these two opts into a single bool instead of doing the || multiple times?

Reply:
Or maybe even a method on options that returns an Option<DeviceVtl> ?

Reply:
I ended up making the inspect stuff not rely on this new flag (which I rename to --get), since we will eventually support a non-GET mode but will still want inspect support.